engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-events.html

```diff
@@ -1,7 +1,7 @@
 <section class="nutupane-sub-section">
   <div bst-container-scroll>
     <div infinite-scroll="eventTable.nextPage()" infinite-scroll-container="'.nutupane-sub-section .container-scroll-wrapper'">
-      <table bst-table="eventTable" class="table table-striped">
+      <table bst-table="eventTable" class="table table-striped" ng-if="eventTable">
```

Comment:
because i had to move it into a promise:  https://github.com/Katello/katello/pull/5695/files#diff-5244a7b283ca094b544c0a858a5f9964R20

Unless you think there is a better way?  Should only be temporary, when we're pointing to the host api directly we will have the host id without having to fetch an object